data-structures-linked-lists/includes.js

```diff
@@ -0,0 +1,17 @@
+/* exported includes */
+
+function includes(list, value) {
+  let listCopy = list;
```

Comment:
This is not a copy. Objects are pass-by-reference. We don't really need a copy anyway.